packages/notifications-system/src/database/services/database.service.ts

```diff
@@ -0,0 +1,64 @@
+import * as mongoDB from "mongodb";
+import * as dotenv from "dotenv";
+import Notification from "../models/notifications";
+
```

Comment:
Here, you should create a new file to create mongoose connection.
notification-system/src/drivers/mongo/connection.js
To create connection I make a function in shared package.
### Steps to be able to use it

1.  update your branch
2. run `yarn bootstarp` in root directory

### Steps to implement it

1. import MongoDriver in your file (notification-system/src/drivers/mongo/connection.js) `const { MongoDriver } = require('@booking-services/shared');`
2. call the function giving a prefix like a parameter 
example about how my team implement it.

```
const { MongoDriver } = require('@booking-services/shared');
const { prefix } = require('../../utils/constants')

module.exports = MongoDriver.createConnection(prefix);
```

**Note:** createConnection return an instance of mongoose so you can use a variable something like this
`const connection = MongoDriver.createConnection(prefix);
module.exports =  connection;
`